node/client/Cargo.toml

```diff
@@ -11,7 +11,7 @@ primitives = { path = "../../core/primitives" }
 storage = { path = "../../core/storage" }
 chain = { path = "../../core/chain" }
 beacon = { path = "../../core/beacon" }
-node-runtime = { path = "../runtime" }
+node-runtime = { path = "../runtime", features = ["test-utils"] }
```

Comment:
also test-utils in prod? seems wrong

Reply:
this is meant to be temporary while things are in flux. here's a ticket that i'll use to track coming back and cleaning everything up https://github.com/nearprotocol/nearcore/issues/137